Array_1_Problem_2.java

```diff
@@ -0,0 +1,81 @@
+//Time Complexity : O(n)
```

Comment:
For 2D array, it would be clearer if you mention time complexity in terms of rows and columns i.e. O(RC) instead of O(N)